src/readers/morphologyHDF5.cpp

```diff
@@ -264,7 +264,7 @@ int MorphologyHDF5::_readSections() {
         const auto& p = vec[i];
         const int& type = p[1];  // Explicit int for Clang (<0 comparison)
 
-        if (type > SECTION_CUSTOM_START || type < 0) {
+        if (type >= SECTION_CUSTOM_START || type < 0) {
```

Comment:
Good catch 👍

Reply:
btw should we also check for `type < 0` ? `0` is not explicitly defined type. The spec does not mention it. @tomdele, @mgeplf ?

Reply:
If not mentioned, let's consider this as invalid.